### DISCUSSION TITLE
Show modified BTTV emotes.

### DISCUSSION THREAD

**Discussion by Pafon22** [Chatterino/chatterino2#4714] [Ideas]
In BTTV, some commands modify your emotes, in addition to being able to combine 2 commands.
Commands:
w! Emote = Makes the emote wide.
h! Emote = Flips the emote horizontally.
v! Emote = Flips the emote vertically.
z! Emote = zero width (overlay).
c! Emote = Color (black and white).

w! v! cuteDog = Wide, upside down dog emote.
But chatterino only displays the text "w! v!" and then the emote.

**Comment by Felanbird** [comment]
Duplicate of #4470
